src/doc/trpl/documentation.md

```diff
@@ -15,7 +15,7 @@ comments":
 // the "link" crate attribute is currently required for rustdoc, but normally
 // isn't needed.
 #![crate_id = "universe"]
-#![crate_type="lib"]
+#![crate_type = "lib"]
```

Comment:
Sorry, I didn't want to create too much noise with trivial pull requests. Do you want me to open a new one for this issue?

(I don't have a clue about correct Rust coding style. But http://aturon.github.io/style/whitespace.html recommends it and I thought it would be good to have the official documentation align with it.)